internal/cluster/validator.go

```diff
@@ -534,9 +536,9 @@ func (v *clusterValidator) networkPrefixValid(c *clusterPreprocessContext) (Vali
 		return ValidationPending, "The Cluster Network CIDR is undefined."
 	}
 	validClusterNetworks := funk.Filter(c.cluster.ClusterNetworks, func(clusterNetwork *models.ClusterNetwork) bool {
-		return clusterNetwork != nil &&
+		return clusterNetwork != nil && (v.skipNetworkHostPrefixCheck(c, clusterNetwork.HostPrefix) ||
```

Comment:
maybe better to take v.skipNetworkHostPrefixCheck(c, clusterNetwork.HostPrefix) outside this line? it is not readable actually.
Maybe we can just set if before and return success in case we should skip it?

Reply:
Thanks, good point. Updated to check the skip first.